libcodechecker/log/log_parser.py

```diff
@@ -791,8 +792,18 @@ def parse_options(compilation_db_entry):
 
     return BuildAction(**details)
 
+def serialize(obj):
```

Comment:
I would put this to the buildaction module, we serialize only that like this right?

Reply:
done